### PR TITLE
Add correct link to generator repo. See #952

### PR DESCRIPTION
**Pull request by sbeleidy**

### CODE REVIEW

File: _includes/docs/0.4.x/gettingstarted.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
     There are several ways you can get the MEAN.JS boilerplate:
   </p>
   <h3>Yo Generator</h3>
-  <p>The generator is still in development. See the <a href="https://github.com/meanjs/generator-meanjs/tree/0.4-dev">Generator Repo</a> for information on how to use the development version.
+  <p>The generator is still in development. See the <a href="https://github.com/meanjs/generator-meanjs">Generator Repo</a> for information on how to use the development version.
```
Comment (ilanbiala) [inline _includes/docs/0.4.x/gettingstarted.html]: Let's change this to something along the lines of the generator is released but we are adding back features from 0.3.3 generator as we convert the code

**Comment by codydaig** [comment]
@sbeleidy I think this PR slipped through the cracks, and now theres a merge conflict. Can you resolve the conflict, and ping us once you do?

**Comment by ilanbiala** [comment]
@codydaig didn't we just merge a PR to fix this?

**Comment by codydaig** [comment]
@ilanbiala This one fixes the wording, the other one only fixed the link.

**Comment by ilanbiala** [comment]
@codydaig for this small fix I don't think it makes sense to wait for the PR, can you just go ahead and make the commit yourself, reference this, and close it once it has been added?

**Comment by codydaig** [comment]
@ilanbiala SGTM

**Comment by codydaig** [comment]
Updated in #1033

**Pull request closed without merging**